packages/components/pie-toggle-switch/test/component/pie-toggle-switch.spec.ts

```diff
@@ -41,7 +42,36 @@ test.describe('Component: `Pie toggle switch`', () => {
         const pieToggleSwitchComponent = await component.locator(componentSelector).isDisabled();
 
         // Assert
-        await expect(pieToggleSwitchComponent).toBe(false);
+        expect(pieToggleSwitchComponent).toBe(false);
+    });
+
+    test('should set `id` to `toggle-switch` on the input element', async ({ mount }) => {
```

Comment:
IMO not sure this test is required because it's not really dynamic. The test below is valid though because it's conditional

Reply:
I sort of agree here, I wasn't going to add this but i felt that as it's tied to the `for` part of the label it might be worth keeping it so it's not messed with or changed in future & not updated in the input element, probably unlikely to happen but added as a belt and braces thing. Happy to remove it tho.

Reply:
Maybe the description is misleading. "should have an attribute ID toggle-switch" or something like this instead?